pandas/util/_decorators.py

```diff
@@ -245,7 +245,7 @@ def wrapper(*args, **kwargs) -> Callable[..., Any]:
     return decorate
 
 
-def doc(*args: Union[str, Callable], **kwargs: str) -> Callable[[F], F]:
+def doc(*args: Union[str, Callable], **kwargs: Any) -> Callable[[F], F]:
```

Comment:
I think this should stay the way it is - can you revert changes to this module? Or is there a particular reason these wouldn't be a str?

Reply:
In this PR they’re bools and ints. Doesn’t make sense to restrict to str, IMO.

Reply:
Ah OK I see your point